deployment/CP02PMCI_Deploy.csv

```diff
@@ -1,100 +1,109 @@
+AR-28,,AR-31,,CP02PMCI-WFP01-05-PARADK000,10,1,2018-04-11T11:58:00,2018-11-05T16:53:00,CGMCP-02PMCI-00010,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.22652,-70.88887,,106,127,
+AR-31,,AR-34,,CP02PMCI-RII01-02-ADCPTG010,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,,CGINS-ADCPTG-19151,40.22665,-70.87826667,,106,126,
+AR-31,,AR-34,,CP02PMCI-SBS01-00-STCENG000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,,CGCON-BSTC01-00017,40.22665,-70.87826667,,0,126,
+AR-31,,AR-34,,CP02PMCI-SBS01-01-MOPAK0000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,,CGINS-MOPAK0-58478,40.22665,-70.87826667,,0,126,
+AR-31,,AR-34,,CP02PMCI-WFP01-00-WFPENG000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,40.22665,-70.87826667,,105,126,
+AR-31,,AR-34,,CP02PMCI-WFP01-01-VEL3DK000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,40.22665,-70.87826667,,105,126,
+AR-31,,AR-34,,CP02PMCI-WFP01-02-DOFSTK000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,40.22665,-70.87826667,,105,126,
+AR-31,,AR-34,,CP02PMCI-WFP01-03-CTDPFK000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,40.22665,-70.87826667,,105,126,
+AR-31,,AR-34,,CP02PMCI-WFP01-04-FLORTK000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,40.22665,-70.87826667,,105,126,
+AR-31,,AR-34,,CP02PMCI-WFP01-05-PARADK000,11,1,2018-11-05T14:53:00,2019-04-18T21:51:00,CGMCP-02PMCI-00011,CGVEH-CWFPCP-99101,CGINS-PARADK-20437,40.22665,-70.87826667,,105,126,
+AR-34,,,,CP02PMCI-RII01-02-ADCPTG010,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,,CGINS-ADCPTG-24542,40.2267,-70.8868,,106,126,
+AR-34,,,,CP02PMCI-SBS01-00-STCENG000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,,CGCON-BSTC01-00040,40.2267,-70.8868,,0,126,
+AR-34,,,,CP02PMCI-SBS01-01-MOPAK0000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,,CGINS-MOPAK0-99999,40.2267,-70.8868,,0,126,sn not in config
+AR-34,,,,CP02PMCI-WFP01-00-WFPENG000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGCON-CWFPCP-10302,40.2267,-70.8868,,105,126,
+AR-34,,,,CP02PMCI-WFP01-01-VEL3DK000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-VEL3DK-00037,40.2267,-70.8868,,105,126,
+AR-34,,,,CP02PMCI-WFP01-02-DOFSTK000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-DOFSTK-02724,40.2267,-70.8868,,105,126,
+AR-34,,,,CP02PMCI-WFP01-03-CTDPFK000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-CTDPFK-00125,40.2267,-70.8868,,105,126,
+AR-34,,,,CP02PMCI-WFP01-04-FLORTK000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-FLORTK-01119,40.2267,-70.8868,,105,126,
+AR-34,,,,CP02PMCI-WFP01-05-PARADK000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.2267,-70.8868,,105,126,
```

Comment:
Reviewed by CDobs. Everything that I could verify in P12 L2 QLR looks good. @snwhite Where can I look to verify the instrument SNs? I don't see those in the config sheet on Vault.

Reply:
Found what I was looking for. Have reviewed the instruments as well.  All good.